packaging/installer/install-required-packages.sh

```diff
@@ -33,19 +32,20 @@ PACKAGES_NETDATA_SENSORS=${PACKAGES_NETDATA_SENSORS-0}
 PACKAGES_NETDATA_DATABASE=${PACKAGES_NETDATA_DATABASE-0}
 
 # needed commands
-lsb_release=$(which lsb_release 2>/dev/null || command -v lsb_release 2>/dev/null)
+lsb_release=$(command -v lsb_release 2> /dev/null)
 
 # Check which package managers are available
-apk=$(which apk 2>/dev/null || command -v apk 2>/dev/null)
-apt_get=$(which apt-get 2>/dev/null || command -v apt-get 2>/dev/null)
-dnf=$(which dnf 2>/dev/null || command -v dnf 2>/dev/null)
-emerge=$(which emerge 2>/dev/null || command -v emerge 2>/dev/null)
-equo=$(which equo 2>/dev/null || command -v equo 2>/dev/null)
-pacman=$(which pacman 2>/dev/null || command -v pacman 2>/dev/null)
-yum=$(which yum 2>/dev/null || command -v yum 2>/dev/null)
-zypper=$(which zypper 2>/dev/null || command -v zypper 2>/dev/null)
+apk=$(command -v apk 2> /dev/null)
+apt_get=$(command -v apt-get 2> /dev/null)
+dnf=$(command -v dnf 2> /dev/null)
+emerge=$(command -v emerge 2> /dev/null)
+equo=$(command -v equo 2> /dev/null)
+pacman=$(command -v pacman 2> /dev/null)
+yum=$(command -v yum 2> /dev/null)
+zypper=$(command -v zypper 2> /dev/null)
```

Comment:
is it really a good thing to have a space after `>` ?

i think `2>/dev/null` is more common then `2> /dev/null`

Reply:
(same for all redirects), imo space is not needed

Reply:
`shfmt` does this. We're pretty much letting tools take care of formatting / code-style for us with agreed upon options. @Ferroin is doing to put up a PR in our docs repo for this.